core/src/main/java/org/hisp/dhis/android/network/common/dto/GeometryDTO.kt

```diff
@@ -29,18 +29,19 @@
 package org.hisp.dhis.android.network.common.dto
 
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
 import org.hisp.dhis.android.core.common.FeatureType
 import org.hisp.dhis.android.core.common.Geometry
 
 @Serializable
 internal data class GeometryDTO(
     val type: String?,
-    val coordinates: String?,
+    val coordinates: JsonElement?,
```

Comment:
The format of this value is variable, it can be a list of double, a list of lists of doubles, or a list of lists of lists of double. There was a custom Serializer in the domain that took care of this. Here, it is modeled as a JsonElement and then it uses the `toString()` method. This class is used in tracker data objects, so will have to double test it there. Also, it will be required to have a `fromDomain()` method to serialize.